tests/testthat/test-spread.R

```diff
@@ -243,3 +239,21 @@ test_that("spread doesn't convert data frames into tibbles", {
   df <- data.frame(x = c("a", "b"), y = 1:2)
   expect_equal(class(spread(df, x, y)), "data.frame")
 })
+
+test_that("spread with fill replaces explicit missing values", {
+  df <- tibble(key = c("a"), value = c(NA))
```

Comment:
Can you please eliminate the needless `c()` here?

Reply:
Sure, also did so in 1 or 2 other places